mms/model_service_worker.py

```diff
@@ -42,6 +40,7 @@ class MXNetModelServiceWorker(object):
     Backend worker to handle Model Server's python service code
     """
     def __init__(self, s_type=None, s_name=None, host_addr=None, port_num=None):
+        os.environ['OMP_NUM_THREADS'] = "1"
```

Comment:
This should be in the config no ?

Reply:
We should not add this here. We should respect whatever is set in env. We can provide a default value if env didn't set it.

Reply:
True. Will set it if its not already set.